text/3463-crates-io-policy-update.md

```diff
@@ -0,0 +1,233 @@
+  "name squatting") without having any genuine functionality, purpose, or
+  significant development activity on the corresponding repository
+- is related to buying, selling, or otherwise trading of package names or any
+  other names on crates.io for money or other compensation
+- impersonates any person or entity, including through false association with
+  crates.io, or by fraudulently misrepresenting your identity or site's purpose
+- is related to inauthentic interactions, such as fake accounts and automated
+  inauthentic activity
+- is using our servers for any form of excessive automated bulk activity, to
+  place undue burden on our servers through automated means, or to relay any
+  form of unsolicited advertising or solicitation through our servers, such as
+  get-rich-quick schemes
+- is other automated excessive bulk activity and coordinated inauthentic
+  activity, such as
+    - spamming
+    - cryptocurrency mining
+- is not functionally compatible with the cargo build tool (for example, a
+  "package" cannot simply be a PNG or JPEG image, a movie file, or a text
+  document uploaded directly to the registry)
+- is abusing the package index for purposes it was not intended
+
+You are responsible for using crates.io in compliance with all applicable laws,
+regulations, and all of our policies. These policies may be updated from time to
+time. We will interpret our policies and resolve disputes in favor of protecting
+users as a whole. The crates.io team reserves the possibility to evaluate each
+instance on a case-by-case basis.
+
+For issues such as DMCA violations, or trademark and copyright infringements,
+the crates.io team will respect the legal decisions of the
+[Rust Foundation](https://rustfoundation.org/) as the official legal entity
+providing the crates.io service.
+
+
+## Package Ownership
+
+crates.io has a first-come, first-serve policy on crate names. Upon publishing a
+package, the publisher will be made owner of the package on crates.io.
+
+If you want to take over a package, we require you to first try and contact the
+current owner directly. If the current owner agrees, they can add you as an
+owner of the crate, and you can then remove them, if necessary. If the current
+owner is not reachable or has not published any contact information the
+crates.io team may reach out to help mediate the process of the ownership
+transfer.
+
+The crates.io team only deletes crates from the registry that do not comply with
+the policies on this document. Crate deletion by their owners is not possible to
+keep the registry as immutable as possible. If you want to flag your crate as
+open for transferring ownership to others, you can publish a new version with
+a message in the README or description communicating to the crates.io support
+team that you consent to transfer the crate to the first person who asks for it:
+
+> I consent to the transfer of this crate to the first person who asks
+> help@crates.io for it.
```

Comment:
With rust-bus, there used to be a co-owner invited to make a similar statement -- such a co-owner (whose name might be "Crate can be transferred at the crates.io team's discretion") would both be less intrusive on README files (which always run a risk of being overloaded) and easier to manage (for it can be added and removed without publishing a new release).

The rust-bus initiative has been closed, but my impression is that that was more a matter of finding a team of suitable people than of the technical implementation.

Reply:
Please note that this section was copied directly from https://crates.io/policies, so there is nothing changing here. I'd prefer for the discussion to focus primarily on the things that were added/changed :)

Reply:
Fine with me, closing marking this as resolved (and unsubscribing unless pinged). It may benefit the further processing of this to visually separate changed and unmodified sections to avoid noise like this.